### PR TITLE
[GITHUB-2] Add "systemd" Support

### PR DESCRIPTION
**Pull request by wojtek-oledzki**

### CODE REVIEW

**Review by lobsterdore — APPROVED** [review]
LGTM :+1:

File: README.md [hunk 1]
```diff
@@ -12,6 +12,9 @@ This role installs Kong on Debian systems, configured to use Postgres for storag
 
 For more information on Kong please visit [getkong](https://getkong.org/).
 
+If you want to test locally using Ubuntu Xenial, add `VAGRANT_BOX=ubuntu/xenial64`
+to `.make` file.
+
```
Comment (quater) [inline README.md]: The `.make` file does not exist by default. I therefore would re-phrase this to make it clear without someone needing to read the Makefile.

```
If you want to test locally by using Ubuntu Xenial, create the `.make` file at the root of this repo and add the line `VAGRANT_BOX=ubuntu/xenial64`.
```

Comment (quater) [reply README.md]: I have reviewed and tested on my laptop locally. Apart from the small doc change it looks good to me.

**Comment by wojtek-oledzki** [comment]
:boom: README updated @quater

**Comment by quater** [comment]
LGTM :+1:

**Pull request merged**